### PR TITLE
build: Webpack production changes

### PR DESCRIPTION
**Pull request by rkmarks**
updated webpack config and build on umd for production. also converted utils to es6 arrow functions

**What kind of change does this PR introduce?**

- [ ] bugfix
- [ ] feature
- [ ] refactoring / style
- [x ] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [x ] Yes, my code is well tested
- [ ] Not relevant

**If relevant, did you update the documentation?**

- [ ] Yes, I've updated the documentation
- [x ] Not relevant

**Summary**

This solve is to implement a production ready component for end users. It also lowers the size of the component for consumption. 

#531 

**Does this PR introduce a breaking change?**

**Other information**
Also to be more ES6 complicit I updated the utils to arrow functions.

### CODE REVIEW

**Comment by rkmarks** [comment]
@okonet new PR w/ cleaner git history :)

File: src/utils/index.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ export const supportMultiple =
     ? 'multiple' in document.createElement('input')
     : true
 
-export function getDataTransferItems(event) {
+export const getDataTransferItems = event => {
```
Comment (okonet) [inline src/utils/index.js]: Any specific reason for switching from `function` to `const`

File: src/utils/index.js [hunk 2]
```diff
@@ -25,19 +25,15 @@ export function getDataTransferItems(event) {
 
 // Firefox versions prior to 53 return a bogus MIME type for every file drag, so dragovers with
 // that MIME type will always be accepted
-export function fileAccepted(file, accept) {
-  return file.type === 'application/x-moz-file' || accepts(file, accept)
-}
+export const fileAccepted = (file, accept) =>
```
Comment (okonet) [inline src/utils/index.js]: Same question here

File: src/utils/index.js [hunk 3]
```diff
@@ -25,19 +25,15 @@ export function getDataTransferItems(event) {
...
+  file.type === 'application/x-moz-file' || accepts(file, accept)
 
-export function fileMatchSize(file, maxSize, minSize) {
-  return file.size <= maxSize && file.size >= minSize
-}
+export const fileMatchSize = (file, maxSize, minSize) =>
```
Comment (okonet) [inline src/utils/index.js]: Let’s revert this file since I don’t see any reason for it to change. Especially not in this PR

**Comment by rkmarks** [comment]
@okonet file reverted

**Comment by rkmarks** [comment]
@okonet ready for merge/review

**Pull request merged**